### PR TITLE
quickstart neuralchat notebook for workshops

### PR DESCRIPTION
**Pull request by kta-intel**
## Type of Change

Documentation

## Description

Consolidated several notebooks (building chatbot, plugins, bf16 optimization, int4 quantization, and finetuning) into a single notebook to be used to introduced NeuralChat at workshops and hackathons. Verified each feature and added some text to make it easier to navigate

## Expected Behavior & Potential Risk

User should be able to install ITREX from source and requirements to run NeuralChat and plugins using the README then follow the notebook cell-by-cell

## How has this PR been tested?

Ran on Intel Developer Cloud ` Intel(R) Xeon(R) Platinum 8480+` instance

## Dependency Change?

N/A

### CODE REVIEW

**Comment by kevinintel** [comment]
Please move the pdf to https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/assets

File: intel_extension_for_transformers/neural_chat/docs/notebooks/workshop/docs/Annual_Report.pdf
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/docs/notebooks/workshop/docs/Annual_Report.pdf:1]: Suggest to provide a link and ask user to download

Comment (kta-intel) [reply intel_extension_for_transformers/neural_chat/docs/notebooks/workshop/docs/Annual_Report.pdf:1]: thanks, I removed the pdf and fixed the retrieval steps in the notebook to access the content

**Comment by kta-intel** [comment]
> Please move the pdf to https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/assets

Thanks, decided to remove the pdf and adjust the steps in the notebook to directly download sample text for retrieval

**Review by hshen14 — APPROVED** [review]

**Pull request merged**